common/read-models/graphql/resolvers.js

```diff
@@ -1,8 +1,8 @@
 export default {
   user: async (store, { id, name }) => {
-    const users = await store.collection('users')
-
-    return id ? await users.findOne({ id }) : await users.findOne({ name })
+    return id
+      ? await store.hget('users_id', id)
+      : await store.hget('users_name', name)
```

Comment:
Unnecessary `await` keyword and `async` keyword is accordingly. It will return 'Promise' anyway. Also better to drop braces because it's only single command inside.